ansys/dpf/core/field.py

```diff
@@ -348,42 +349,39 @@ def unit(self, value):
         
     @property
     def dimensionality(self):
-        """The Dimensionality represents the shape of an elementary
-        data contains in the Field.
+        """Dimensionality represents the shape of the elementary
+        data contained in the field.
```

Comment:
```suggestion
        """Dimensionality represents the shape of the elementary data contained in the field.
```
When possible, let's try to keep the sort description on the first line only.